docs/videos/healthcare-and-life-sciences/hcls-blueprint.yaml

```diff
@@ -0,0 +1,356 @@
+    source: modules/scripts/startup-script
+    settings:
+      runners:
+      - $(spack.install_spack_deps_runner)
+      - $(spack.install_spack_runner)
+      - type: shell
+        destination: setup_gromacs.sh
+        content: |
+          #!/bin/bash
+          source /apps/spack/share/spack/setup-env.sh
+          spack env activate gromacs
+          chmod -R a+rwX /apps/spack/var/spack/environments/gromacs
+          mkdir -p /apps/gromacs
+          chmod a+rwx /apps/gromacs
+          cd /apps/gromacs
+          wget --no-verbose https://ftp.gromacs.org/pub/benchmarks/water_GMX50_bare.tar.gz
+          tar xzf water_GMX50_bare.tar.gz
+
+      - type: data  # rethink
+        destination: /apps/gromacs/submit_gromacs.sh
+        content: |
+          #!/bin/bash
+          #SBATCH -N 2
+          #SBATCH --ntasks-per-node 30
+          #SBATCH -p compute
+
+          # Size can be 0000.65  0000.96  0001.5  0003  0006  0012  0024  0048  0096  0192  0384  0768  1536  3072
+          # Type can be 'pme' or 'rf'
+
+          source /apps/spack/share/spack/setup-env.sh
+          spack env activate gromacs
+
+          # Check that gmx_mpi exists
+          which gmx_mpi
+          cd $SLURM_SUBMIT_DIR
+          cp /apps/gromacs/water-cut1.0_GMX50_bare/1536/* .
+          scontrol show hostnames ${SLURM_JOB_NODELIST} > hostfile
+          gmx_mpi grompp -f pme.mdp -c conf.gro -p topol.top -o input.tpr
+          mpirun -n 60 -hostfile hostfile -ppn 30 gmx_mpi mdrun -notunepme -dlb yes -v -resethway -noconfout -nsteps 4000 -s input.tpr
+
+      - type: shell  # TODO: consider making bool option
+        destination: shutdown.sh
+        content: |
+          #!/bin/bash
+          if [ ! -f /etc/block_auto_shutdown ]; then
+                  touch /etc/block_auto_shutdown
+                  shutdown -h +1
+          fi
+
+  - id: spack-builder
+    source: modules/compute/vm-instance
+    use: [network-install, appsfs-install, spack-builder-startup]
+    settings:
+      name_prefix: spack-builder
+      machine_type: c2-standard-16  # TODO Consider bumping up, left low for quota
+
+- group: cluster
+  modules:
+
+  - id: network1
+    source: modules/network/pre-existing-vpc
+
+  - id: homefs
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.homefs_server_ip)
+      remote_mount: nfsshare
+      local_mount: /home
+      fs_type: nfs
+
+  - id: appsfs
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      server_ip: $(vars.appsfs_server_ip)
+      remote_mount: nfsshare
+      local_mount: /apps
+      fs_type: nfs
+
+  - id: bucket-software
+    source: modules/file-system/pre-existing-network-storage
+    settings:
+      remote_mount: $(vars.bucket_name_software)
+      local_mount: /user_provided_software
+      fs_type: gcsfuse
+      mount_options: defaults,_netdev,implicit_dirs
+
+  - id: bucket-input
```

Comment:
You downloaded the data to the /apps storage. Why adding a bucket-input here? 
I understand you may want to show you could have input and output buckets, but since you are actually not using it, isn't it more confusing?

Reply:
We intended to use the input bucket for the demo. I have updated the scripts to stage data to the input bucket.